CollAction/Controllers/ProjectsController.cs

```diff
@@ -364,12 +365,35 @@ public ViewResult StartInfo()
         }
 
         [HttpGet]
-        public async Task<JsonResult> GetTileProjects(int? categoryId, int? locationId)
+        public async Task<JsonResult> GetTileProjects(int? categoryId, int? locationId, string statusId)
         {
-            var displayTileProjectViewModels = await _service.GetTileProjects(Url, p => p.Status != ProjectStatus.Hidden && p.Status != ProjectStatus.Deleted
+            Expression<Func<Project, bool>> projectExpression = (p => p.Status != ProjectStatus.Hidden && p.Status != ProjectStatus.Deleted
```

Comment:
I'm not sure if ternary statements get optimized to sql well. Maybe check in a query tracer if it gets optimized?

If not, I'd pull the parameter check outside the query body, like:

if (categoryId != null)
    projectExpression = projectExpression.AndAlso..